app/views/admin/categories/index.html.erb

```diff
@@ -16,7 +16,7 @@
                             </tr>
                             <% @categories.each do |category| %>
                             <tr>
-                                <td class="index-text-content"><%= category.title %></td>
+                                <td class="index-text-content"><%= link_to category.title, admin_words_path(category), class: "index-text-content-title"%></td>
```

Comment:
Since we update the routes.rb. You need to change the path.
admin_category_words_path(category)